Authentication/SignUp/View/SignupView.swift

```diff
@@ -199,147 +187,147 @@ internal final class SignupView: UIView, SignupViewType, NibLoadable {
+            passwordErrorsView.isHidden = true
         } else {
             // To trigger the Valid's didSet.
             let valid = passwordTextFieldValid
             passwordTextFieldValid = valid
         }
     }
     
-    private func passwordVisibleWasSet() {
+    func passwordVisibleWasSet() {
         toggleVisibility(passwordTextFieldOutlet, visible: passwordVisible,
                          visibilityButton: passwordVisibilityButtonOutlet, visibilityTitle: passwordVisibilityButtonTitle)
     }
     
-    private func passwordConfirmationTextFieldValidWasSet() {
+    func passwordConfirmationTextFieldValidWasSet() {
         if !passwordConfirmationTextFieldSelected {
             let color: CGColor
             if passwordConfirmationTextFieldValid {
-                color = delegate.colorPalette.textfieldsNormal.CGColor
-                passwordConfirmationErrorsView.hidden = true
+                color = delegate.colorPalette.textfieldsNormal.cgColor
+                passwordConfirmationErrorsView.isHidden = true
             } else {
-                color = delegate.colorPalette.textfieldsError.CGColor
-                passwordConfirmationErrorsView.hidden = false
+                color = delegate.colorPalette.textfieldsError.cgColor
+                passwordConfirmationErrorsView.isHidden = false
             }
             pswdConfirmTextFieldAndButtonViewOutlet.layer.borderColor = color
         } else {
-            passwordConfirmationErrorsView.hidden = true
+            passwordConfirmationErrorsView.isHidden = true
         }
     }
     
-    private func passwordConfirmationTextFieldSelectedWasSet() {
+    func passwordConfirmationTextFieldSelectedWasSet() {
         if passwordConfirmationTextFieldSelected {
-            passwordConfirmTextFieldOutlet.layer.borderColor = delegate.colorPalette.textfieldsSelected.CGColor
-            passwordConfirmationErrorsView.hidden = true
+            passwordConfirmTextFieldOutlet.layer.borderColor = delegate.colorPalette.textfieldsSelected.cgColor
+            passwordConfirmationErrorsView.isHidden = true
         } else {
             // To trigger the Valid's didSet.
             let valid = passwordConfirmationTextFieldValid
             passwordConfirmationTextFieldValid = valid
         }
     }
     
-    private func confirmationPasswordVisibleWasSet() {
+    func confirmationPasswordVisibleWasSet() {
         toggleVisibility(passwordConfirmTextFieldOutlet, visible: passwordConfirmationVisible,
                          visibilityButton: passwordConfirmVisibilityButtonOutlet, visibilityTitle: confirmPasswordVisibilityButtonTitle)
     }
     
-    private func toggleVisibility(textField: UITextField, visible: Bool, visibilityButton: UIButton, visibilityTitle: String) {
+    func toggleVisibility(_ textField: UITextField, visible: Bool, visibilityButton: UIButton, visibilityTitle: String) {
```

Comment:
`func toggleVisibility(in textField: UITextField, visible: Bool, visibilityButton: UIButton, visibilityTitle: String)` ?